### DISCUSSION TITLE
Biticino living now - Test connessione Gateway K4500c e interruttore

### DISCUSSION THREAD

**Discussion by Giampy72** [indomus/forum#1216] [GENERALE - Discussioni generiche]
Ho installato in casa il gateway in oggetto con tutti gli interruttori , prese e termostati della stessa linea.
Non riuscendo ad integrarlo in HA e non volendo fare test sull'impianto esistente visto che in casa ci vivo, ho acquistato un secondo Gateway per fare i test e ho capito come integrarlo in HA facendo ripetuti tentativi e ripetuti reset fino a raggiungere un installazione stabile. Nei prossimi giorni proverò con altri componenti visto che per il test ho integrato il classico interruttore oltre al Gateway.  Mi restano da provare le prese smart , il comando tapparelle e i termostati Smarttether 2

Tra le varie guide , la cosa che non leggo mai ben chiara, è come abbinare il Gateway . Se lo si abbina all'app ufficiale bticino, HA non lo vede piu. Solo in una circostanza forse casuale, sono riuscito a far convivere il gateway su entrambe le applicazioni ( Home Controll e Home Assistant ) .  Se riesco a ricreare la stessa situazione con il kit di test , scrivo qui tutti i passaggi. 
Allo stato attuale, sono riuscito ad integrare il gateway e un interruttore sacrificando però l'App Bticino che a mio avviso , pur rispondendo un po lentamente ai comandi istantanei, da un grado di affidabilità per gli scenari automatici legati a cose semplici tipo accensione o spegnimento luci ad un particolare orario e ovviamente l'utilissima gestione carichi.  
HA su raspberry non so fino a che punto possa dare stabilità nel tempo ... devo testarlo visto che ho messo in piedi tutto solo da un mesetto. Sicuramente piu interessante la possibilità di creare tante interazioni e integrazioni con oggetti di marca differente usando la programmazione.

Voi come avete integrato Living Now ?

**Comment by maxcanna** [comment]
Personalmente ti consiglio di eliminare il gateway originale. Compra un coordinator zigbee e poi usa zigbee2mqtt per esporre gli interruttori ad home assistant. A quel punto potrai creare tutte le automazioni che vorrai

**Comment by cobradodo** [comment]
Se devi usare i comandi tapparelle tieni il gateway originale e comandale con integrazione HomeKit, a meno che non ti interessa nulla della percentuale di apertura e chiusura tapparelle.
Per quanto riguarda il resto invece sposo l'idea di @maxcanna metti tutto tramite zigbee e via... 

Io per ora ho solo i comandi tapparelle e mi sono trovato costretto, per il motivo spiegato prima, ad acquistare il gateway e usare app control, integrando con HomeKit, ma semmai acquisterò altri frutti eviterò app ufficiale

**Comment by Giampy72** [comment]
Probabilmente alla fine opterò per una chiavetta conbee. 
Se sto cercando di insistere con i test è solo per il fatto che ero riuscito ad integrare il gateway e tutti i componenti ( interruttori , prese , tapparelle e termostati ) senza scollegarli dall'app originale di Bticino Home + Controll e senza resettare nulla .

**Reply by maxcanna** [reply]
Ti consiglio un coordinator usb custom made, non stare a prendere conbee

**Comment by Giampy72** [comment]
Mi sembrava quella piu facile da installare. 
cosa mi consigli ?
Perchè escludete subito l’integrazione del gateway K4500c?

**Reply by maxcanna** [reply]
Non aprire un nuovo thread per ogni risposta 😅 detto questo, ti consiglio di usare un coordinator zigbee al posto del gateway originale per un discorso di espandibilita futura del sistema. Allo stesso coordinator puoi facilmente collegare dispositivi zigbee di produttori diversi. Col gateway originale non si può. Quanto al coordinator ti consiglio quello prodotto da @Gio-dot

**Reply by gio-dot** [reply]
> Mi sembrava quella piu facile da installare. cosa mi consigli ? Perchè escludete subito l’integrazione del gateway K4500c?

La facilità di installazione è la stessa per tutti i tipi di chiavetta. Ciò che fa la differenza è il software con il quale le integri in Homeassistant, ed al momento il migliore e più compatibile è Zigbee2mqtt. Detto questo la Conbee2 non è la scelta migliore in assoluto (in quanto a copertura e supporto in Zigbee2mqtt). I chip zigbee migliori al momento sono i cc2652p come quello che utilizzo nella mia chiavetta Z-Bee Duo.

**Reply by Giampy72** [reply]
Ok quindi seplicemente dovrei eliminare i deviatori dall’App bticino, che equivale a resettarli ( led rosso) , e a questo punto la chiavetta li rileva in HA. Giusto?
Gia questo sarebbe un bel risultato per le automazioni. Lascerei le prese sul K4500c per il controllo dei consumi e gestione carichi visto che nel quadro ho il modulo bticino che fa tutto.

**Reply by Giampy72** [reply]
Però come si spiega che ero riuscito ad integrate tutto senza rinunciare al gateway ?

**Reply by maxcanna** [reply]
Esatto, la procedura è questa. Comunque ti conviene portare tutto su z2m se fai il passaggio. Anche il controllo carichi. Tu hai usato un'altra integrazione, probabilmente homekit

**Reply by Giampy72** [reply]
esatto con autodiscovery ha rilevato automaticamente i componenti usando l'integrazione homekit .  
ma a differenza di quanto si legge nelle guide , dove si parla di usare homekit a patto di scollegare il gateway dalle App e resettare il tutto ( incluso il gateway stesso ) , nel mio caso homeassistant aveva rilevato tutto in automatico quindi di fatto riuscendo a far convivere home assistant con l'app ufficiale bticino.  Ora però non riesco piu a ricreare quella situazione che a mio avviso non era poi cosi malvagia visto che ad oggi non ho altri componenti di altre marche zigbee da dover integrare. ( per adesso :) )
ad ogni modo , dove si può acquistare la Z-bee Duo.  I link che vedo mi portano su uno store in $  .  Non esiste lo shop in Italia visto che lo sviluppatore è italiano ?

**Reply by snakuzzo** [reply]
Lo store è quello...ed è internazionale. Vai sereno con l'acquisto. Se hai dubbi puoi contattare direttamente @Gio-dot

**Reply by snakuzzo** [reply]
Se hai intenzione di utilizzare l'hub Bticino puoi contattare @sdomotica che dovrebbe permetterti di integrare facilmente il tutto su HA con il suo addon/custom component

**Reply by Giampy72** [reply]
proverò ad eliminare il gateway... 
l'unico dubbio che mi sorge è legato alle prese smart:   per resettarle da app bticino basta rimuoverle e tornano con spia rossa come se le avessi resettate manualmente.  
Come si resettano con l'integrazione HA ? 
alcune prese sono dietro elettrodomestici pesanti ed è quindi impossibile resettarle manualmente se dovesse andare storto qualcosa .

**Reply by essegi65** [reply]
Non voglio assolutamente passare avanti a nessuno, riporto solo quella che è la mia configurazione che al momento non mi crea problemi. Io ho installato il GW bticino (gestito anche tramite app Bticino) a cui ho collegato solo deviatori e a sua volta collegato a HA tramite Homekit Controller mentre per i comandi presa al momento sono collegati tramite Zigbee Home Assistant (con cui riesco a vedere anche i consumi, cosa che con il GW bticino non viene rilevata).

**Reply by Giampy72** [reply]
grazie essegi65, 
per me ogni informazione è utile .  
Io con l'app bticino vedo i consumi delle prese smart. Anzi aggiungendo sucessivamente il modulo gestione carichi, sempre dall'App Bticino ho configurato la priorità per la disattivazione degli elettrodomestici in caso di consumi troppo alti.
Infatti pensavo di fare esattamente l'opposto di come l'hai configurata tu, ovvero, avrei lasciato la gestione dei carichi e prese al gateway e avrei integrato le luci ad HA tramite chiavetta ma per il semplice fatto che non riesco piu ad integrarle direttamente usando homikit controller.  
Riesci a darmi qualche dritta su come usare homekit controller per l'integrazione ? 
A me risponde sempre che non trova nessun componente.  L'unico modo per far apparire il gateway su HA è dissociarlo dall'App Bticino e resettare tutti i componenti. Come dicevo all'inizio ho comprato un gateway solo per fare i test ( cosa che potrebbe essere utile a chi, come me, ha questo problema di integrazione senza dover acquistare una nuova chiavetta ) e per adesso ho un deviatore smart sempre adibito al test.  Preferisco non maneggiare sull'impianto di casa prima di essere sicuro al 100%  . Quindi se qualcuno ha dubbi sul da farsi e vuole proporre qualche test , posso farli io .  Ho anche la possibilità di testare un comando tapparelle .

**Reply by essegi65** [reply]
Mi sembra di ricordare di aver letto il procedimento su una discussione nel vecchio forum di indomus. Seguendo la guida li ha trovati subito alla prima volta sia il GW che 1 comando presa. successivamente ho aggiunto dei deviatori, sempre prima su homecontrol Bticino che poi sono stati rilevati su HA da homekit control. Tieni presente che se lo hai associato, il GW, una volta ad homekit controller e poi lo hai eliminato devi iniziare da capo disassociandolo anche da l'app homecontrol di bticino. Se non ricordo male questo è quanto feci la prima volta, scusa ma vado a memoria sono passati quasi 2 anni.

**Reply by Giampy72** [reply]
esatto. è proprio questo il mio problema. Ero riuscito anche io ad integrare tutto in modo automatico ( deviatori, prese, termostati ) ma non ricordo come ho fatto . Pensando fosse semplice ho dovuto ricreare la macchina home assistant e , nonostante i vari tentativi , non riesco piu a farlo integrare. Le sto provando tutte.  Non contento , visto che sono testardo , ho preso un gateway nuovo di pacca e sto cercando di configurarlo a banco , su una macchina HA nuova.  Ma niente da fare , non riesco a ricreare quella situazione iniziale, che con tanta semplicità mi aveva fatto apprezzare anche l'integrazione Home control .  
Sembra che funzioni solo dissociandolo dall'App Bticino. eppure anche tu mi confermi che convivono contemporaneamente senza problemi.

**Reply by Giampy72** [reply]
Oggi mi è apparso uno dei 7 termostati. Lo avevo rimosso dall’app, resettato, inserito nuovamente nell’app bticino. E dopo svariati giorni, senza aver fatto nulla , mi è apparso nelle integrazioni HA.
Sono confuso 🤔 
O meglio… lo avevo provato ad inserire anche nell’App Casa Apple e dopo averlo rimosso solo da quest’ultima mi è apparso su HA. Quindi allo stato attuale il termostato lo vedo su HA e su app bticino da dove ho programmato lo scenario termico.

**Reply by Stomodine** [reply]
Mi associo anche io a Giampy72 : fatto delle prove con una VM in virtual box e comandavo senza problemi sia il gateway ticino che il tahoma somfy. ho comprato un raspberry e non riesco a vedere piu' nulla (anche se ho eliminato l'homekit controller dalla macchina virtuale).

@Giampy72 il fatto che tu sia riuscito a vederlo solo dopo averlo inserito e tolto in homekit apple è normale, viene detto anche sul sito di home assistant (sorry ma non trovo la pagina al momento). il problema sorge nel momento in cui ,come me, non si hanno prodotti apple

**Reply by Giampy72** [reply]
Posso provare con un altro termostato. Lo associo ad Apple Home ( oltre che Home+ control bricino ) e lo rimuovo subito dopo , cosi almeno capiamo se funziona .  I termostati credo che siano piu semplici visto che sono wifi e non zigbee ( almeno credo ) . 
ad ogni modo se vuoi che faccio qualche test con il kit nuovo , chiedi pure ed io eseguo. Tutte le idee sono utili per arrivare ad un risultato “non casuale” ma da poter rifare in qualsiasi momento nel caso , come me, ci sia necessita di riconfigurare HA. Non si puo ogni vota resettare il gateway perdendo tutte le configurazioni delle stanze e dei dispositivi solo per tentare di vederli in HA senza nessuna certezza di risultato.

**Reply by Stomodine** [reply]
"Non si puo ogni vota resettare il gateway perdendo tutte le configurazioni delle stanze e dei dispositivi solo per tentare di vederli in HA senza nessuna certezza di risultato."

Esattamente...mi piacerebbe capire se esiste un modo per disaccoppiare i dispositivi direttamente da HA ma non ho trovato info in rete.

per il test potresti vedere se lanciando l'homekit controller ti trova il gateway nuovo (che però,secondo me, devi prima associare all'app home control)

**Reply by Giampy72** [reply]
Per adesso ho fatto questa prova:
- kit nuovo ( o resettato )
- Associo deviatori quindi classica procedura da led rossi a verdi
- Dato che il gateway non conosce ancora la rete wifi , uso app bticino per associare al wifi 
- Ma a questo punto HA non vede il gateway perche probabilmente è gia associato all’app bticino

seconda prova 
- kit nuovo ( o resettato ) 
- Associo deviatori quindi led da rossi a verdi
- Invece di usare l’app bticino per associare il gateway al wifi, apro le reti dalle impostazioni wifi del telefono o del computer e vedo una rete wifi Legrand che sarebbe il gateway. 
- selezionandolo mi chiede se voglio associarlo ad una rete wifi casalinga e lo faccio
- A questo punto mi appare in HA e sarebbe ok , se non fosse che non posso piu abbinarlo all’App bticino

in merito a dissociare i dispositivi da HA , non credo si possa fare ottenendo un reset quindi led rosso, come avviene con l’app bticino. Non sarebbe un grosso problema con gli interruttori ma lo è invece con le prese che di solito sono dietro elettrodomestici ingombranti e quindi irraggiungibili e impossibili da resettare manualmente. Di conseguenza occorre un modo affidabile per fare il reset via software e solo l’app bticino lo permette , per questo non voglio eliminarla .

**Reply by Giampy72** [reply]
ok ci sono riuscito , in modo questa volta metodico e nn casuale .
il segreto è 
1- configurare prima tutto con l'app Home + Control di Bticino  ( probabilmente quindi non serve resattare nulla )
2- con un dispositivo Apple ( iphone ) creo una nuova casa e in automatico l'app importa tutti i componenti presenti nell App Bticino ( sarà un passaggio solo temporaneo ma necessario )
3- elimino il gateway legrand dall' App Casa di Apple ( iphone ) appena creata
4- finalmente appare il gateway Legrand direttamente su HA - Integrazioni  (senza bisogno di fare aggiungi integrazione)
5- premo configura sull'integrazione appena apparsa su HA e inserisco il numero identificatico del gateway ( vicino al QrCode )
6- tutto appare in automatico senza bisogno di far nient'altro 

purtroppo però bisogna avere un iphone, anche solo in prestito, per il passaggio numero 2 e 3.

ho provato anche a riavviare HA e spegnere l'impianto di test , e tutto continua a funzionare su entrambe le piattaforma ( bticino e HA )
esempio , accendo luce su Bticino , mi aggiorna lo stato su HA

tengo acceso e sotto test per un po e vediamo se funziona tutto in modo stabile.
prossimo step , aggiungere una presa e un interruttore tapparelle

**Reply by Stomodine** [reply]
riguardo al punto 2 , come fa l'app di apple a importare tutto in automatico?

te lo chiedo perchè ho recuperato un ipad ma non trova nessun dispositivo homekit

inoltre come faccio a resettare il gateway bticino?

**Reply by Giampy72** [reply]
scusa , hai ragione, ho dato per scontato che anche tu avessi Iphone. 
in pratica la App Casa Apple vede automaticamente quanto installato nell'App Bticino home+control installata nello stesso telefono.
potresti trovare un vecchio ipad usato e dedicarlo , ad esempio , a pannello di comando dell'intera casa, come ho fatto io.
ho reciclato un vecchio ipad, installato tutte le app della casa ( bticono , home assistant, reloink per le telecamere ecc ecc. ) 
in realta quando sarà tutto a regime , puoi usare l'ipad direttametne a parete, con sempre aperto HA.

**Comment by Stomodine** [comment]
Ok grazie ma come faccio a resettare il gateway ticino?

Il ven 24 dic 2021, 19:38 Giampy72 ***@***.***> ha scritto:

**Reply by Giampy72** [reply]
Per resettare il gateway c’è l’apposito tastino sulla parte frontale ma io aspetterei a resettarlo.
Farei prima la prova partendo dal punto 2.

**Comment by Giampy72** [comment]
Istruzioni per far convivere contemporaneamente gli smarttether bticino sia sull’app ufficiale che con HA:
1 - resettare il termostato 
2- abbinare il termostato all’app bticino con la sua procedura classica
3 - aprire l’app Casa di Apple , dove probabilmente ci sarà anche qui il termostato installato
4 - rimuoverlo dall’App Casa Apple
5 - aprire HA e vedrete il termostato nelle integrazioni quindi premere comfigura
6 - inserire il codice del termostato vicino al qrcode

provare a partire dal punto 3 . Potrebbe bastare se avete un’ installazione pulita e non avete mai fatto altri tentativi.
in questo modo si puo usare il termostato con entrambe le applicazioni ( HA e Bticino )

**Reply by Giampy72** [reply]
Mi è arrivata una chiavetta Conbee II e la sto testando.  Semplicità disarmante.  resta però il problema del reset fisico dei dispositivi e quindi, se per gli interruttori è semplice, per le prese dietro gli elettrodomestici è improponibile. 
Quindi promossa per gli interruttori da inserire negli scenari.

**Reply by Stomodine** [reply]
aggiungo la mia esperienza: se il gateway è già stato associato a qualche utenza homekit (che sia apple oppure l'homekit controller di HA) non c'è verso di assegnarlo ad un altro "master". L'unico modo è resettarlo ai dati di fabbrica.

Dopo aver fatto il reset e reimpostato dall'app home+control, il gateway compare senza problemi nella lista dei device rilevati da homekit (anche qui,sia apple che controller di HA).

@Giampy72 i passaggi 3 e 4 del tuo penultimo messaggio non sono necessari; ne approfitto per un'altra domanda: visto che stai usando conbee, mi confermi che per associare ad esempio gli interruttori a conbee devo prima resettarli (se già associati al gateway ticino)

PS: un'altra cosa sul reset del gateway: il comando master entra-esci viene disaccoppiato e quindi va riassegnato (io ce l'ho fatta ma non ho ben capito come; credo si debba tenere premuto il reset del gate fino a quando si accende verde fisso e poi premere brevemente uno dei pulsanti master)

**Reply by Giampy72** [reply]
il punto 3 e 4 ho visto nei miei test , che servono per vedere i termostati sia su app bticino che su HA. Probabilmente non servono se li installi direttamente su HA.
Però riprovo per maggior sicurezza.

Gli interruttori bisogna resettarli ma non solo , bisogna anche premere una seconda volta il tasto reset per metterli in modalità di paring. Per questo motivo temo che sia poco fattibile con le prese posizionate dietro gli elettrodomestici, nel senso che non basta disassociarli dall’applicazione ( led rosso ).

Per quanto riguarda l’applicazione bticino e HA ( senza uso di conbee ii ) , possono coesistere insieme a patto di resettare bene tutto e quindi partire da zero, passando da Apple Casa , prima di HA , per poi rimuoverlo subito sempre da Apple Casa.  Quindi si puo fare anche senza conbee ii , ma devo dire che quest’ultima soluzione è molto piu reattiva. L’unico lato negativo è perdere la possibilità di gestire tutto anche da bticno come ad esempio il comando entra esci.

**Reply by Stomodine** [reply]
per le prove che ho fatto io (con gateway e tahoma somfy) confermo che non c'è bisogno di passare per apple casa

PS:dove hai comprato la conbee?

**Reply by Giampy72** [reply]
quindi mi confermi che stai usando contemporaneamente l'app Bticino home+control e Home Assistant senza problemi ?  Quindi rispetto alla mia lista di 6 punti , salti il punto 3 e 4 , e ti appare ugualmente il gateway su HA ?

la conbee l'ho presa su Amazon

io ho riprovato ancora ma se non passo da Apple Casa aggiungendo ed eliminando il GW, non mi appare su HA. 
tu esattamente cosa fai ?

**Reply by Stomodine** [reply]
confermo che mi è bastato resettare gateway e tahoma per vedermeli comparire su HA

**Reply by Giampy72** [reply]
Puoi spiegarti meglio ? Tahoma centra qualcosa ? Immagiano sia un altro tipo di gateway per altri dispositivi ma rimanendo su bticino cosa hai fatto oltre resettarlo ? Lo vedi anche su app bticino ?
Che step fai per configurare tutto?

**Reply by Stomodine** [reply]
Esatto...Tahoma è un gateway tipo quello ticino per le tende Somfy

Ad ogni modo, dopo aver resettato il ticino, l'ho associato all'app home+control. Dopodichè me lo sono trovato nei device rilevati da home assistant

**Reply by Giampy72** [reply]
Caspita ho provato piu volte e appena resetto e abbino al wifi appare in Home Assistant ma appena lo abbino a biticino App , sparisce da HA. 
io non riesco proprio. Forse perche io lo faccio da iphone mentre tu lo fai da android ? Non saprei . So solo che pur facendolo tante volte non riesco proprio a vedere il GW in HA se non passo da Casa Apple

**Reply by Giampy72** [reply]
Sei riuscito a verificare come resettare le prese di corrente ?

**Reply by Stomodine** [reply]
Dovevo verificare il reset delle prese? scusa ma con tutti questi post non ricordo :)

**Reply by Giampy72** [reply]
te ne sarei grato. piu che altro per non rischiare di abbandonare dietro grossi elettrodomestici le prese connesse dovuto al fatto di non poterle resettare manualmente.  vorrei essere sicuro che basti un reset software . sicurametne con app bticino basta rimuoverle e si resettano ( credo come per i deviatori ) , ma da HA non si può semplicemente rimuovendo il dispositivo e quindi occorrerebbe intervenire fisicamente spostando lavatrice , asciugatrice, forni ecc diventa scomodo . Per questo motivo forse è meglio avere entrambe le app funzionanti in parallelo.

**Reply by Stomodine** [reply]
lo farò non appena mi arriva la conbee...tempo una settimana credo.

Ci riaggiorniamo

**Reply by Giampy72** [reply]
ciao , con la combee ti confermo che bisogna premere fisicamente due volte il tasto reset sul deviatore ( non solo per resettarlo ma anche per accoppiarlo ) e probabilmente vale lo stesso sulla presa ( quindi per queste ultime mi sembra improponibile )  , e inoltre devi rinunciare all'applicazione ufficiale bticino.  Io l'ho presa solo per qualche deviatore sul quale voglio fare degli scenari particolari. 
Mentre io intendevo il reset senza conbee, quindi nella situazione originaria, dove vogliamo far convivere HA con Home+control Bticino.

**Reply by Stomodine** [reply]
ciao

ti confermo che per accoppiare la presa al conbee devi per forza resettarlo da tastino e perdi l'integrazione con home control. Da quello che ho visto basta che una volta acceso con spia rossa attivi l'autodiscovery in zigbee2mqtt e lo associa senza ulteriori azioni (discorso diverso invece per il F20T60A,con questo devi premere due volte)

**Reply by Giampy72** [reply]
Ok grazie.
Forse pero’ la cosa migliore è integrare le prese su bticino gateway dato che ho anche il modulo din per la gestione dei carichi, e lasciare ad HA con o senza conbee la gestione degli scenari luci, integrandoli di fatto con altri dispositivi … es.  allo scattare dell’allarme accendere le luci con una sequenza a piacimento per simulare la presenza in casa , magari attivando anche l’irrigazione del giardino per creare un fattore di disturbo in esterno.
Non riesco a immaginare uno scenario in HA per le prese …. Forse solo a livello statistico per grafici di consumi. Ovviamente l’inventiva aiuta a trovarne i giusti utilizzi. Ma la cosa piu importante e prioritaria e’ per me la gestione dei carichi e preferisco sia su un sistema stand alone che non dipenda da alcun computer

**Reply by Stomodine** [reply]
solo una precisazione: due prese una volta associate alla conbee si sono spente da sole mentre altre 3 ho dovuto ripremere il tastino reset
